src/components/Modal/Modal.tsx

```diff
@@ -48,17 +52,21 @@ const slideInBottom = keyframes({
   },
 });
 
-const containerStyle = (theme: Theme) =>
-  typeStyle({
-    maxHeight: '70vh',
-    background: colorShades(theme.background)[1],
-    zIndex: 1001,
-    top: '10vh',
-    boxShadow: shadow('2X', theme),
-    overflowY: 'auto',
-    width: '50vw',
-    animation: `${slideInBottom} 0.2s cubic-bezier(0.250, 0.460, 0.450, 0.940) both`,
+const modalStyle = (theme: Theme) => {
+  return stylesheet({
+    container: {
+      maxHeight: '70vh',
+      background: colorShades(theme.background)[1],
+      zIndex: 1001,
+      top: '10vh',
+      boxShadow: shadow('2X', theme),
```

Comment:
You need to remove these from all the places or else shadows added by elevation will have no effect on it.

Reply:
Do check it for other components..

Reply:
Done.